exercises/collatz-conjecture/canonical-data.json

```diff
@@ -44,7 +44,7 @@
       "input": {
         "number": 0
       },
-      "expected": { "error": "Only positive numbers are allowed" }
+      "expected": { "error": "Only positive integers are allowed" }
```

Comment:
Since this is technically a breaking change and not a bug fix, as some language _will_ need to change their tests based on this output, you probably want to _copy_ the entire test case, and use `reimplements` and _deprecate_ this one.

cc @ErikSchierboom @petertseng

Reply:
I copied the test case and used `reimplements`. I can't see anything about deprecating tests - how do I do that?

Reply:
Ah. A test case is deprecated if it is reimplemented by a new test case :)